src/hQuery/Node.php

```diff
@@ -1217,7 +1220,6 @@ public function prev()
     public function rewind()
     {
         reset($this->ids);
-        return $this->current();
```

Comment:
Is this causing any warnings?
I'd prefer to keep this return in order not to break someone's code relying on this.

Reply:
Sorry for the late reply I've been a little busy recently and no It doesn't cause any warnings for me but I agree it would be better keep the return. 

I also ran the tests using `test.sh` script and it's showing 1 deprecation for some reason. I'm still unfamiliar with the current test setup and not sure how to turn them into exceptions:

```
Runtime:       PHP 8.2.9

...

Time: 00:03.902, Memory: 120.89 MB

OK, but there were issues!
Tests: 25, Assertions: 285, Deprecations: 1.
```

Reply:
`test.sh` is basically running `phpunit tests/` inside a docker container with a given PHP version. You could run the same command manually using the version of PHP you have available in your system.
I'll try to get rid of that deprecation warning.